src/enc.h

```diff
@@ -2092,7 +2092,7 @@ static void doloop (int argc, char *argv[]) {
 			}
 		}
 
-		putenv("ULTRAVNC_DSM_HELPER_LOOP_SET=1");
+		setenv("ULTRAVNC_DSM_HELPER_LOOP_SET", "1", 1);
```

Comment:
According to http://www.greenend.org.uk/rjk/tech/putenv.html, setenv is superior, but might not be available. Can you add an autotools check for it and #ifdef accordingly?